memtier_benchmark.cpp

```diff
@@ -104,7 +104,7 @@ static void config_print(FILE *file, struct benchmark_config *cfg)
         "expiry_range = %u-%u\n"
         "data_import = %s\n"
         "data_verify = %s\n"
-        "verify_only = %s\n"
+        "verify_get_only = %s\n"
```

Comment:
@tomerr-lb @sashas-lb the verify_only arg is used by the data_verify option of memtier. By changing it's name we break this feature, hence it will make it harder to contribute code if we still plan to. 
Maybe in-order to keep the crc_verify and the data_verify features separated we should add a new argument "crc_verify_get" or something of sorts and restore the use of verfiy_only arg to be used only with data_verify.
אני פשוט חושבת שאנחנו מערבבים שמחה בשמחה וזה יכול ליצור לנו כאב ראש בעתיד...

Reply:
mmm @tomerr-lb  she has a point .. your call guys ..